### ISSUE TITLE
Mk3.5 connect shows einsy rambo not buddy board

### ISSUE THREAD

**Issue by cozmo14047** [prusa3d/Prusa-Link-Web#489]
Hi, when u connect the mk3.5 to p4usa connect and go to settings it says einsy rambo serial number instead of buddy board serial number

**Comment by ondratu** [comment]
Hi, that is not (or can't be) exactly possible. Did you use same printer token as you used in MK3 for Connect communication? If not, what you see is buddy board serial number.

**Comment by cozmo14047** [comment]
> Hi, that is not (or can't be) exactly possible. Did you use same printer token as you used in MK3 for Connect communication? If not, what you see is buddy board serial number.

Yes i see the buddy board serial number but it says seinsy rambo sn: (buddy board sn) 
![image](https://github.com/prusa3d/Prusa-Link-Web/assets/134814825/b624514d-75c9-408d-aca6-58e3b1e48afe)

**Comment by ondratu** [comment]
I see, bad label, thanks for report!

**Comment by cozmo14047** [comment]
all good, sorry if i was confusing before

> I see, bad label, thanks for report!
